linter/whitelists/en.txt

```diff
@@ -130,6 +130,7 @@ Preview
 Privacy
 production
 Projects
+Protocol
```

Comment:
I don't think this whitelist entry is necessary, since you are not using that word (with upper case `p`) in your meta data (and it would be a correct English word anyway).